addons/udes_stock/views/stock_picking.xml

```diff
@@ -111,6 +111,43 @@
                 <attribute name="string">Cancel Pick</attribute>
                 <attribute name="attrs">{'invisible': [('state', 'in', ('draft', 'done', 'cancel'))]}</attribute>
             </xpath>
+            
+            <!-- Add button to show all fields, by default hidden fields will not be shown -->
+            <xpath expr="//button[@name='action_confirm']" position="before">
+                <button name="action_detailed_view" string="Detailed View" type="object"/>
+            </xpath>
+
+            <xpath expr="//div[@name='button_box']" position="inside">
+                <button class="oe_stat_button"
+                        name="open_first_pickings"
+                        icon="fa-building-o"
```

Comment:
This is always the toughest part even though it is minor. :)

The icon doesn't match well with pickings imo, could we try `fa-exchange` - this icon is used for Product Moves on the product view, and pickings essentially mean products are moved.

Could be something to discuss with product, I believe we should have access to most of these icons for v14 at least: https://fontawesome.com/v4/icons/

Reply:
fa-exchange doesnt look bad, if you dont like it than will speak with PO, but i think is ok